src/sql/workbench/contrib/notebook/browser/cellViews/textCell.component.ts

```diff
@@ -78,6 +82,7 @@ export class TextCellComponent extends CellView implements OnInit, OnChanges {
 	private markdownRenderer: NotebookMarkdownRenderer;
 	private markdownResult: IMarkdownRenderResult;
 	public previewFeaturesEnabled: boolean = false;
+	public turnDownService = new TurndownService({ 'emDelimiter': '*' });
```

Comment:
@chlafreniere What are your thoughts on using this library?

@halerankin What made you decide to start using this here now?

Reply:
@Charles-Gagnon Please ask Maddy about turndown as she implemented it.

Reply:
@MaddyDev Comments here?